### DISCUSSION TITLE
The rope fired by the fleshgun cannot stick to the submarine anymore.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#13844] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As the title said, I think it's caused by the latest update regarding the rope.
check the following code:
https://github.com/FakeFishGames/Barotrauma/blob/f240bd83b2b8004d38fc40951bdfc724bef2d5f1/Barotrauma/BarotraumaShared/SharedSource/Items/Components/Rope.cs#L177-L192
Now, the existence of the rope depends on the user of the projectile that it is bound to.
So, if its user is null, the rope will snap.
The fleshgun is controlled by AI rather than a character, so the rope fired by it cannot be attached to anything.

### Reproduction steps

1. Enter sub editor
2. Create a fleshgun with `AutoOperate` enabled and link it to a fleshgun ammo sack
3. Test
4. Spawn a sub above the fleshgun
5. Waiting for the fleshgun to attack the submarine 

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I [created a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13851), we'll look into it!

**Comment by Jasontti** [comment]
Fix tested and no issues found.
Closing discussion and ticket.
